docs/misc/changelog.rst

```diff
@@ -53,6 +53,8 @@ Documentation:
 - Fix ``clip_range`` docstring
 - Fix duplicated parameter in ``EvalCallback`` docstring (thanks @tfederico)
 - Added example of learning rate schedule
+- Added SUMO-RL as example project (@LucasAlegre)
+- Fix docstring of AtariWrapper which was not inside of __init__ method (@LucasAlegre)
```

Comment:
Please revert this, docstrings are right after the constructor for the automatic documentation

Reply:
But it was giving an error when running "make-doc". Now it is like all other classes.

Reply:
That should not be the case, the doc is building at least with continuous integration, see https://github.com/DLR-RM/stable-baselines3/runs/1533347883

See current doc too: https://stable-baselines3.readthedocs.io/en/master/common/atari_wrappers.html

Reply:
The error I found before was because it was written "frame_skip::" instead of "frame_skip:" (with only one ":").
Now I fixed the docs of all the other classes in the file too.